src/components/Loading/Loading.scss

```diff
@@ -96,26 +96,26 @@
       border-right: 13px solid transparent;
     }
   }
-
-  
-  
 }
 
 
 .loading--sm {
-  display: block;
+  display: inline-flex;
   z-index: 1031;
 
+  .msgsmall{
```

Comment:
`msg--sm`?

Reply:
This is not the same div container for `msgsmall` and `msg`, so a _modifier_ does not seem to be suitable in this case.